### PR TITLE
add brainstorming meeting 2 notes and related spec files

### LINKED ISSUE #18: Write brainstorming meeting notes
# Overview
Write and push the brainstorming meeting notes (for Friday 05/03) in /admin/meetings. Store any image/mindmap/... we made during the meeting in specs/brainstorm

## Frontend (UI/UX)
N/A

## Backend (CRUD operations)
N/A

## Acceptance Criteria
- [x] Meeting notes in /admin/meetings
- [x] Assets in specs/brainstorm

### PR DESCRIPTION
**Pull request by sadsoap**
## Tracking Info

Resolves #18 

## Changes

added brainstorming meeting 2 notes and related spec files

## Testing

N/A
## Confirmation of Change

Added brainstorm meeting notes from 05/03 to admin/meetings, and added the updated feature mindmap and user stories to specs/brainstorm.

### CODE REVIEW

File: admin/meetings/050324-brainstorm
Comment (RamtinTJB) [inline admin/meetings/050324-brainstorm:1]: Make sure the file extension is .md for markdown

**Review by RamtinTJB — CHANGES_REQUESTED** [review]
Great work! Just fix the extension for the markdown file and we are good to go

**Review by RamtinTJB — APPROVED** [review]
Thanks!

**Comment by lindseyrapp** [comment]
Thanks for doing this!

**Pull request merged**